sample.env

```diff
@@ -2,6 +2,9 @@
 # (Required) The IP address of Docker host machine which can be found by running "ifconfig"
 # Hint: look for "inet addr:" within "eth0" or "en0" for OSX
 DOCKER_HOST_IP=""
+MAVEN_GITHUB_TOKEN=
```

Comment:
Can we link to documentation on how to generate this in the sample.env. Something like this:
```.env
# Required for building the deduplicator. Documentation: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
```
Also can you add a section to the `readme.md` that documents the environmental variables and background of the deduplicator application?

Reply:
Also should this be in the deduplicator portion of the `sample.env` as it is the only component that uses it?

Reply:
This could be shared in the future, but for now its fine to put in the deduplicator section. Fixed.